addons/refuel/script_component.hpp

```diff
@@ -3,7 +3,7 @@
 #include "\z\ace\addons\main\script_mod.hpp"
 
 // #define DRAW_HOOKS_POS
-// #define DEBUG_MODE_FULL
+// #define DEBUG_ENABLED_REFUEL
```

Comment:
```suggestion
// #define DEBUG_MODE_FULL
```
this is template in all ACE addons

Reply:
I reverted this change.